src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerBookmarks.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FunctionComponent } from 'react';
+import upperFirst from 'lodash/upperFirst';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { RootState } from 'src/store';
+import { PreviouslyViewedObject } from 'src/content/app/browser/track-panel/trackPanelState';
+import { BrowserTrackStates } from 'src/content/app/browser/track-panel/trackPanelConfig';
+import { updateTrackStatesAndSave } from 'src/content/app/browser/browserActions';
+import { closeTrackPanelModal } from 'src/content/app/browser/track-panel/trackPanelActions';
+import { closeDrawer } from 'src/content/app/browser/drawer/drawerActions';
+import { getActiveGenomePreviouslyViewedObjects } from 'src/content/app/browser/track-panel/trackPanelSelectors';
+
+import styles from './DrawerBookmarks.scss';
+
+type StateProps = {
+  previouslyViewedObjects: PreviouslyViewedObject[];
+};
+
+type DispatchProps = {
+  updateTrackStatesAndSave: (trackStates: BrowserTrackStates) => void;
+  closeTrackPanelModal: () => void;
+  closeDrawer: () => void;
+};
+export type DrawerBookmarksProps = StateProps & DispatchProps;
+
+const DrawerBookmarks: FunctionComponent<DrawerBookmarksProps> = (
```

Comment:
no need to use the FunctionComponent type

Reply:
Removed it